### ISSUE TITLE
Graql: key constraint

### ISSUE THREAD

**Issue by grabl** [graknlabs/grakn#3860]
_This issue was originally posted by @haikalpribadi  on 2018-04-10 18:57_.

Define cardinality restriction of an entity-attribute by declaring the constraint keyword after the attribute declaration in an entity type declaration.
For now, just implement the key cardinality constraint. But when we have multiple types of cardinality constraints, it should look like below.

```
define
person sub entity,
    has username key,
    has firstname,
    has lastname min=1 max=2,
    has email unique;
````

**Comment by vjcharles** [comment]
I apologize for the comment here* but in testing graql I've noticed the key keyword does not appear to work as intended in the latest version 1.4.3 as discussed here https://discuss.grakn.ai/t/what-does-key-actually-do/786/4.
Issue noticed by using the console to load in two separate files with entity data via `./graql console --keyspace entities --file THIS_PATH/entity-data-FIRST.gql` then `./graql console --keyspace entities --file THIS_PATH/entry-data-SECOND.gql`.

* both linux firefox and chromium browsers aren't able to sign in to your Discourse platform via the 'gmail' option.

**Comment by haikalpribadi** [comment]
The problem you highlighted has been resolved in 1.5.0, @vjcharles. And this issue has actually been already implemented.
